ollama/_client.py

```diff
@@ -90,11 +90,8 @@ def __init__(
       base_url=_parse_host(host or os.getenv('OLLAMA_HOST')),
       follow_redirects=follow_redirects,
       timeout=timeout,
-      headers={
-        'Content-Type': 'application/json',
-        'Accept': 'application/json',
-        'User-Agent': f'ollama-python/{__version__} ({platform.machine()} {platform.system().lower()}) Python/{platform.python_version()}',
-      }.update(headers or {}),
+      # Lowercase all headers to ensure override
+      headers={k.lower(): v for k, v in {**(headers or {}), 'Content-Type': 'application/json', 'Accept': 'application/json', 'User-Agent': f'ollama-python/{__version__} ({platform.machine()} {platform.system().lower()}) Python/{platform.python_version()}'}.items()},
```

Comment:
this should be formatted for readability with each key/value should have its own line